.github/workflows/wiki.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Wiki-related'
+
+on:
+  push:
```

Comment:
No reason for on push here, the cron schedule is the thing. However, you could add a "workflow-dispatch" trigger to enable updating manually